### ISSUE TITLE
Tool Belt Uninteractible

### ISSUE THREAD

**Issue by CJulien** [tgstation/tgstation#25475]
Problem Description: While wearing a toolbelt containing the items Coil, jaws of life, screwdriver, experimental welding tool, multitool, wirecutters, the tool belt suddenly became completely uninteractable, you cannot open its interface nor remove it from the beltslot

What did you expect to happen: To interact with the tool belt

What happened instead: The tool belt was completely nonresponsive

Why is this bad/What are the consequences: All of the items would've been lost, had an admin not been there to do an alldrop, briefly allowing the toolbelt to be interacted with again.

Steps to reproduce the problem: Equip toolbelt with things, wait indeterminate amount of time for toolbelt to break.

Server: Sybil

Revision: Server revision compiled on: 2017-03-24
24ec2dad32f8f783301f8b50aa362c72be86518f 

Possibly related stuff (which gamemode was it? What were you doing at the time? Was anything else out of the ordinary happening?): Wish I could but I aint got shit

**Comment by BeeSting12** [comment]
#25259?

**Comment by Jalleo** [comment]
Maybe maybe not we need more info. @optimumtact

**Comment by optimumtact** [comment]
no way to tell after the fact unfortunately, it seems like a similar issue but it could be unrelated.

Really needs an admin to inspect the toolbelt and review it's loc.

**Comment by Cyberboss** [comment]
Seriously, if this happens again, please ahelp it

**Comment by Cyberboss** [comment]
@lzimann Can we close this in favor of #25814 ?
